src/Compilers/CSharp/Portable/Binder/Binder.cs

```diff
@@ -554,7 +554,7 @@ internal void ReportDiagnosticsIfObsolete(DiagnosticBag diagnostics, Conversion
         /// 
         /// NOTE: The return value reflects obsolete-ness, not whether or not the diagnostic was reported.
         /// </returns>
-        private static ThreeState ReportDiagnosticsIfObsoleteInternal(DiagnosticBag diagnostics, Symbol symbol, SyntaxNodeOrToken node, Symbol containingMember, BinderFlags location)
+        internal static ThreeState ReportDiagnosticsIfObsoleteInternal(DiagnosticBag diagnostics, Symbol symbol, SyntaxNodeOrToken node, Symbol containingMember, BinderFlags location)
```

Comment:
I'm not 100% sure whether the `Internal` suffix conveys accessibility or layering here.
Although the method accessibility changed, the layering is still there, with a `ReportDiagnosticsIfObsolete` calling it.
I'm thinking to leave this name as-is.